<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePageNavigator</class>
 <widget class="QWidget" name="SimplePageNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="previousPageLabel">
     <property name="toolTip">
      <string>上一页</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="previousSeparate" native="true">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="leftPagesWidget" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="leftSeparateLabel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="centerPagesWidget" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="rightSeparateLabel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightPagesWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="nextSeparate" native="true">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nextPageLabel">
     <property name="toolTip">
      <string>下一页</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>45</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
